build.sbt

```diff
@@ -13,7 +21,36 @@ lazy val `akka-persistence-jdbc` = project
           "akka.persistence.jdbc.util.DefaultSlickDatabaseProvider#lambda#1.apply")),
     // special handling as we change organization id
     mimaPreviousArtifacts := ProjectAutoPlugin.determineMimaPreviousArtifacts(scalaBinaryVersion.value))
-  .settings(AutomaticModuleName.settings("akka.persistence.jdbc"))
+
+lazy val docs = project
+  .enablePlugins(ProjectAutoPlugin, AkkaParadoxPlugin, ParadoxSitePlugin, PreprocessPlugin, PublishRsyncPlugin)
+  .settings(
+    name := "Akka Persistence JDBC",
```

Comment:
Should these settings be moved to a plugin in `project` as others?

Reply:
These will never be relevant for multiple subprojects, it works well to have everything at one place.
Some of it could possibly move to the akka-paradox plugin some day.